packages/client/webapp/src/assets/css/style.css

```diff
@@ -15,6 +15,11 @@
 	--un-color-accent: #2D8E2Dff;
 	--un-color-accent-light: #92E192ff;
 	--un-color-accent-lighter: #E4FAE4ff;
+
+	--un-z-index-sidebar: 50;
+	--un-z-index-navbar: 100;
+	--un-z-index-notification: 300;
+	--un-z-index-modal: 500;
```

Comment:
Looks fine to me, but I know @bencodrington has some opinion and/or suggestions about z-indices in WM project because they are relative to the context and not meant as an absolute measure.

Reply:
Yes, I would love to have opinions on the matter.
In this case, we can just add them if you have a notification or sidebar within a modal with
```css
.modal-notification {
    z-index: calc(var(--un-z-index-modal) + var(--un-z-index-notification));
}
```

Reply:
Sold!